config/initializers/apipie.rb

```diff
@@ -19,6 +19,7 @@
     :operatingsystem_families => Operatingsystem.families.join(", "),
     :providers => ComputeResource.providers.join(', '),
     :default_nic_type => InterfaceTypeMapper::DEFAULT_TYPE.humanized_name.downcase,
+    :template_kinds => -> { TemplateKind.scoped.map(&:name).join(", ") },
```

Comment:
This is broken. 'scoped' is deprecated. http://stackoverflow.com/questions/18198963/with-rails-4-model-scoped-is-deprecated-but-model-all-cant-replace-it

:template_kinds => -> { TemplateKind.where(nil).map(&:name).join(", ") },